arbitrum-docs/node-running/quickstart-running-a-node.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Quickstart: Run different types of Arbitrum nodes"
+description: Learn more about what type of ARb node one needs to run.
+author-objective: Build a quickstart that helps readers understand why they might want to run a specific type of an Arbitrum node.
+reader-audience: Moderately-technical readers who are familiar with command lines, but not Ethereum / Arbitrum infrastructure
+reader-task: Run a node with minimal effort and maximum understanding
+content-type: quickstart
+---
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md'; 
+
+<PublicPreviewBannerPartial />
+
+This quickstart is for who users/developers who want to start running any type of Arbitrum nodes. It makes no assumptions about your prior experience with Ethereum, Arbitrum, or Solidity. Familiarity with command lime expected. 
```

Comment:
Not sure if this line is needed. The page doesn't really have instructions on it. I like the next paragraph as an Introduction.